src/main/java/nl/thehyve/ocdu/validators/patientDataChecks/DateOfBirthPatientDataCheck.java

```diff
@@ -0,0 +1,58 @@
+package nl.thehyve.ocdu.validators.patientDataChecks;
+
+import nl.thehyve.ocdu.models.OCEntities.Subject;
+import nl.thehyve.ocdu.models.OcDefinitions.MetaData;
+import nl.thehyve.ocdu.models.errors.ValidationErrorMessage;
+import org.apache.commons.lang3.StringUtils;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+import java.util.Date;
+
+/**
+ * Created by bo on 6/15/16.
+ */
+public class DateOfBirthPatientDataCheck implements PatientDataCheck{
+    @Override
+    public ValidationErrorMessage getCorrespondingError(int index, Subject subject, MetaData metaData) {
+
+        String ssid = subject.getSsid();
+        String commonMessage = getCommonErrorMessage(index, ssid);
+
+        DateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy", Locale.ENGLISH);
+        dateFormat.setLenient(false);
+
+        ValidationErrorMessage error = null;
+        String dob = subject.getDateOfBirth();
+        if(!StringUtils.isBlank(dob)) {
+            if(dob.length() == 4) {//BIRTH YEAR
+                try{
+                    int currentYear = Calendar.getInstance().get(Calendar.YEAR);
+                    int birthYear = Integer.valueOf(dob);
+                    if(birthYear > currentYear) {
+                        error  = new ValidationErrorMessage(commonMessage+"Birth year format is invalid.");
```

Comment:
if you use CMD+ALT+ L it will format nicely your code. It will help you avoid missing spaces between operators and variables for instance.